src/test/java/com/scottlogic/FilterTests/AuthorFilterTest.java

```diff
@@ -0,0 +1,31 @@
+package com.scottlogic.FilterTests;
+
+import com.scottlogic.UserPost;
+import com.scottlogic.filter.FilterAuthor;
+import com.scottlogic.utils.UserPostFilterData;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+public class AuthorFilterTest {
```

Comment:
AuthorFilterTest/FilterAuthor

Reply:
As per the comments above, I thought it would be FilterAuthorTest under /come/scottlogic/filter/FilterAuthorTest.java - unless I misunderstood what you meant sorry?

Reply:
It looks right now :)